.github/workflows/mcp-tests.yml

```
@@ -85,6 +85,7 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           working-directory: ./mcp_server
+          update-sdk-lock: 'false'
```

actionlint (1.7.12)

[error] input "update-sdk-lock" is not defined in action "Setup Python with uv" defined at "./.github/actions/setup-python-uv". available inputs are "install-dependencies", "python-version", "uv-version", "working-directory"

(action)